src/Compilers/VisualBasic/Portable/CodeGen/EmitStatement.vb

```diff
@@ -1255,7 +1255,7 @@ OtherExpressions:
                 constraints:=constraints,
                 isDynamic:=False,
                 dynamicTransformFlags:=Nothing,
-                isSlotReusable:=synthesizedKind.IsSlotReusable(_optimizations))
+                isSlotReusable:=synthesizedKind.IsSlotReusable(_ilEmitStyle!= ILEmitStyle.Release))
```

Comment:
I guess ```!``` is treated as a type character, but it is a type character for Single and _ilEmitStyle is not Single. It feels strange that compiler doesn't report any error for this code.